tests/Unit/Message/Payload/LtiMessagePayloadTest.php

```diff
@@ -160,4 +164,15 @@ public function testGetValidatedRoleCollection(): void
 
         $this->assertEquals(3, $this->subject->getValidatedRoleCollection()->count());
     }
+
+    public function testGetEndAssessmentReturnDefaultValue(): void
+    {
+        $payload = $this->builder
+            ->reset()
+            ->buildMessagePayload($this->createTestRegistration()->getPlatformKeyChain());
```

Comment:
WDYT of adding another test case when the claim is set to be a non-true value?

Reply:
it's already done in the setUp(), where the claim is set to true, is that what you mean ?

Reply:
I mean, a case when the claim is set, but has a `false` value.

Reply:
done